src/components/MainComponents/Continent.js

```diff
@@ -271,4 +271,11 @@ const Continent = ({ selectedContinent, navSearch }) => {
   );
 };
 
+Continent.propTypes = {
+  selectedContinent: PropTypes.shape({
+    selected: PropTypes.string.isRequired,
+  }).isRequired,
+  navSearch: PropTypes.string.isRequired,
```

Comment:
- [ ] **[OPTIONAL]** Kindly look at the images below. Your search functionality always returns an even number of country cards even if the actual result count is an odd number. In the images, the returned count is odd in number (5 and 1) but I can see an unnecessary search result (circled/marked). It would be nice to remove the unused card.

![aminehl-search](https://user-images.githubusercontent.com/36542986/142432214-312e6e72-2f41-4011-9377-553c3fb89260.jpg)

Reply:
that's just the background but i'll remove it thanks for the feedback appreciated it